### DISCUSSION TITLE
Templates creation

### DISCUSSION THREAD

**Discussion by Gottarocket** [Kunzisoft/KeePassDX#1361] [Q&A]
Yes I have read wiki and created group and did add entry but not sure where to enter type {generator:true}
I just want a credit card template. Help appreciated. Better if you can share a gif or video on how to add those

**Comment by J-Jamet** [comment]
The easiest way is still to generate the default templates of the application.
I guess you are using a database already created, so you just have to activate the templates in the settings of your database in "Settings- Database settings - Templates usage" (you can deactivate the setting and reactivate it if you want to recreate the default template group).
